pkg/tuf/client.go

```diff
@@ -44,6 +44,12 @@ import (
 const (
 	// DefaultRemoteRoot is the default remote TUF root location.
 	DefaultRemoteRoot = "https://tuf-repo-cdn.sigstore.dev"
+	// DefaultRemoteGCSBucket is the name of the GCS bucket that holds sigstore's public good production TUF root
+	DefaultRemoteGCSBucket = "sigstore-tuf-root"
```

Comment:
Do you want to make these private constants to avoid library consumers using them?